@xen-orchestra/web-core/lib/components/UserLink.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <RouterLink v-if="route" :class="{ disabled }" :to="route" class="user-link is-link typo p3-medium">
+    <UserLogo class="logo" />
+    <slot />
+  </RouterLink>
+  <span v-else :class="{ disabled }" class="user-link typo p3-medium">
+    <UserLogo class="logo" />
+    <slot />
+  </span>
+</template>
+
+<script lang="ts" setup>
+import UserLogo from '@core/components/UserLogo.vue'
+import { type RouteLocationRaw } from 'vue-router'
+
+defineProps<{
+  route?: RouteLocationRaw
+  disabled?: boolean
+}>()
+</script>
+
+<style lang="postcss" scoped>
+/* COLOR VARIANTS */
+.user-link {
+  --color: var(--color-purple-base);
+  --border-color: var(--color-purple-base);
+
+  &.is-link {
+    .logo {
+      /* Override the default color of UserLogo,
+       to apply hover, active and disabled styles */
+      --border-color: inherit;
+    }
+
+    &:is(:hover, .hover, :focus-visible) {
+      --color: var(--color-purple-d20);
+      --border-color: var(--color-purple-d20);
+    }
+
+    &:is(:active, .pressed) {
+      --color: var(--color-purple-d40);
+      --border-color: var(--color-purple-d40);
+    }
+
+    &:is(:disabled, .disabled) {
```

Comment:
`span` or `a` doesn't allow the `disabled` attribute.

```suggestion
    &.disabled {
```